packages/editor/src/client/index.tsx

```diff
@@ -76,8 +77,21 @@ const onDOMContentLoaded = async () => {
     return forward(operation)
   })
 
+  const authErrorLink = onError(({graphQLErrors, networkError, operation, forward}) => {
+    if (graphQLErrors) {
+      graphQLErrors.forEach(({message, locations, path, extensions}) => {
+        console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
```

Comment:
Please comment the `console.log` out. It's good for debugging but we don't need that in production.